plugins/processors/execd/execd_test.go

```diff
@@ -50,9 +50,18 @@ func TestExternalProcessorWorks(t *testing.T) {
 		e.Add(m, acc)
 	}
 
-	acc.Wait(1)
-	require.NoError(t, e.Stop())
-	acc.Wait(9)
+	go func(t *testing.T) {
+		for {
+			n := acc.NMetrics()
+			if n == 10 {
+				if err := e.Stop(); err != nil {
```

Comment:
stop can only be called once.

Reply:
It is, isn't it? It's an endless loop that upon reaching the desired metrics count - 10 - stops the process.
Note the return below.

Reply:
ah, yes, sorry, I misread it.